File: tools/python-execute-eval/agent-solutions/claude-sonnet-4-6/notes-crud.py
async def main(params):
    ops = params.get("ops", [])
    for op in ops:
        if op["op"] == "write":
            await kody.call("notes.write", {"id": op["id"], "text": op["text"]})
        elif op["op"] == "remove":
            await kody.call("notes.remove", {"id": op["id"]})
    return await kody.call("notes.list", {})

File: tools/python-execute-eval/agent-solutions/claude-sonnet-4-6/notes-crud.mjs
import { kody } from 'kody:runtime'

export default async function main(params) {
	const ops = params.ops ?? []
	for (const op of ops) {
		if (op.op === 'write') {
			await kody.call('notes.write', { id: op.id, text: op.text })
		} else if (op.op === 'remove') {
			await kody.call('notes.remove', { id: op.id })
		}
	}
	return kody.call('notes.list', {})
}
